src/index.js

```diff
@@ -1 +1,37 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] [OPTIONAL] It can be good to fix errors brought in by linters instead of disabling them.